spring-boot-project/spring-boot/src/main/java/org/springframework/boot/info/SpringInfo.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2012-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.info;
+
+import org.springframework.boot.SpringBootVersion;
+import org.springframework.core.SpringVersion;
+import org.springframework.core.env.Environment;
+import org.springframework.util.ObjectUtils;
+
+/**
+ * Information related to Spring projects.
+ *
+ * @author Jonatan Ivanov
+ * @since 3.3.0
+ */
+public class SpringInfo {
+
+	private final SpringFrameworkInfo framework;
+
+	private final SpringBootInfo boot;
+
+	public SpringInfo(Environment environment) {
+		this.framework = new SpringFrameworkInfo(environment);
+		this.boot = new SpringBootInfo();
+	}
+
+	public SpringFrameworkInfo getFramework() {
+		return this.framework;
+	}
+
+	public SpringBootInfo getBoot() {
+		return this.boot;
+	}
+
+	/**
+	 * Information about Spring Framework.
+	 */
+	public static class SpringFrameworkInfo {
+
+		private final String version;
+
+		private final String[] profiles;
+
+		public SpringFrameworkInfo(Environment environment) {
+			this.version = SpringVersion.getVersion();
+			this.profiles = ObjectUtils.isEmpty(environment.getActiveProfiles()) ? environment.getDefaultProfiles()
+					: environment.getActiveProfiles();
+		}
+
+		public String getVersion() {
+			return this.version;
+		}
+
+		public String[] getProfiles() {
+			return this.profiles;
```

Comment:
I used to put this under a separate `environment` key, please let me know what you think about placing this here.

Reply:
While it technically is a Spring Framework concept, I don't know how many users know this. I would move it up one level:

```json
"spring": {
  "profiles": ["default"]
}
```

or maybe, like you said, add a new key, e.g. `environment` for it.

Reply:
I added it to the `spring` section, I think I like it there more too.
I'm ok with a new `environment` section too but for me I never had anything else that this in it since the `/env` endpoint is also there.